docs/index.md

```diff
@@ -17,25 +17,26 @@ DESCRIPTION
 Determines whether a DANS bag is valid according to the DANS BagIt Profile v0 or v1. If the bag
 does not specify what version of the profile it claims to comply with, v0 is assumed. This module has
 both a command line and an HTTP interface. The command line interface is documented in the
-[ARGUMENTS](#arguments) section below. The HTTP interface is documented in the <a href="../api.html" target="__blank">Swagger UI</a>.
+[ARGUMENTS](#arguments) section below. The HTTP interface is documented in the <a href="api.html" target="__blank">Swagger UI</a>.
 
 ARGUMENTS
 ---------
 
     Options:
 
-          --aip                      Validate as AIP (instead of as SIP)
+          --aip                      Validate the bag(s) as AIP (instead of as SIP)
           --bag-store  <arg>         The bag store to use for deep validation
       -f, --response-format  <arg>   Format for the result report (default = text)
+          --sipdir                   Validate bags inside directories of trailing argument
       -h, --help                     Show help message
       -v, --version                  Show version of this program
-    
-     trailing arguments:
-      bag (not required)   The bag to validate
-    
-    Subcommand: run-service - Starts EASY Validate Dans Bag as a daemon that services HTTP requests
-      -h, --help   Show help message
-    ---
+      
+      trailing arguments:
+       bag (not required)   The bag to validate or in case of '--sipdir': directory with bags in sub directories
+      
+      Subcommand: run-service - Starts EASY Validate Dans Bag as a daemon that services HTTP requests
+        -h, --help   Show help message
+      ---
```

Comment:
```suggestion
      ---
      
EXAMPLES
--------

    easy-validate-dans-bag --aip bagDir
    Validates the bag inside the bagDir as an AIP, without deep validation (i.e. checking the bag store in case of Is-Version-Of in the bag-info.txt)
      
    easy-validate-dans-bag --aip -f json --sipdir sipsDir
    Validates the bags inside the sipsDir as AIPs, without deep validation. 
    The valid bags are untouched, the non-valid bags are moved to a new `sipsDir-nonvalid-timestamp` directory. 
    A JSON file with the same name lists the reasons for the violations  
            
```